src/lib/helpers.js

```diff
@@ -100,3 +100,28 @@ export const parseRelayOptions = options => {
 export const removeNulls = object => {
   Object.keys(object).forEach(key => object[key] == null && delete object[key]) // eslint-disable-line eqeqeq, no-param-reassign, max-len
 }
+// Validates Relay cursor or page/size pagination
+export const validatePagingParams = args => {
+  const { page, size, first, last } = args
+  if ((page && !size) || (size && !page)) {
+    throw new Error("Must specify both a page and size param.")
+  }
+  if (page && size && (first || last)) {
+    throw new Error(
+      "Must specify either page/size or cursor args, but not both."
+    )
+  }
+}
+// Returns `pagingOptions` and `offset`.
+// `pagingOptions` are valid for Gravity API V1.
+// `offset` can be used for connection slicing.
+export const parsePagingParams = args => {
+  const { page, size } = args
+  const pagingOptions = page && size ? { page, size } : parseRelayOptions(args)
+  const offset = page && size ? (page - 1) * size : pagingOptions.offset
+
+  return { pagingOptions, offset }
+}
+export const totalPages = (total, size) => {
+  return Math.ceil(total / size)
+}
```

Comment:
‘helpers’ or ‘utils’ is a bit of a smell to me, especially when there’s multiple functions related to the same topic. How about breaking this out into ‘pagination something’?

Reply:
You mean move some of these into a pagination helper file? Sure, this file currently is just a catch-all for little utility methods such as this, I can move them though.